content/docs/introduction/roadmap.md

```diff
@@ -24,11 +24,10 @@ in Prometheus and provide it via an experimental API endpoint.
 The OpenMetrics working group is developing a new standard for metric exposition.
 We plan to support this format in our client libraries and Prometheus itself.
 
-### Backfill time series
+### Support for retroactive rule evaluations		
 
-Backfilling will permit bulk loads of data in the past. This will allow for
-retroactive rule evaluations, and transferring old data from other monitoring
-systems.
+Backfilling will permit bulk loads of data in the past. This will allow for		
```

Comment:
We already permit bulk loads of data from the past.

Reply:
Have rephrased.